packages/components-native/.eslintrc.js

```diff
@@ -6,5 +6,18 @@ module.exports = {
         allow: ["**"],
       },
     ],
+    "react/forbid-elements": [
```

Comment:
thoughts on dropping this on eslintrc under root and targeting both components-native and Mobile.stories.tsx?

```js
module.exports = {
  ...
  overrides: [
    {
      files: ["packages/components-native/**", "docs/**/Mobile.stories.tsx"],
      rules: {
        ...
      },
    },
  ]
}

Reply:
@darryltec updated it